plugins/catalog/src/routes.ts

```diff
@@ -25,6 +25,6 @@ export const rootRoute = createRouteRef({
 });
 export const entityRoute = createRouteRef({
   icon: NoIcon,
-  path: '/catalog/:kind/:optionalNamespaceAndName/',
+  path: '/catalog/:kind/:optionalNamespaceAndName/:selectedTabId/*',
```

Comment:
hmm how does this work when there is no selected tab ID? like, doesn't this path REQUIRE that all of those things are specified

Reply:
Yup, it does. I've changed it to point to /overview where i found it linking to the entity page. Would be a good thing to have a default as well.